content/docs/data-api/get-started.md

```diff
@@ -0,0 +1,181 @@
+
+</TabItem>
+
+<TabItem>
+You can enable the Neon Data API for your project by sending a `POST` request to our `/data-api` endpoint.
+
+Typically, you can call `/data-api` with no parameters at all. If your branch only has one database, we'll select it automatically. If there are multiple databases, you'll need to specify which one you want to use, as shown below:
+
+Example `curl` request:
+
+```bash shouldWrap
+curl --location 'https://console.neon.tech/api/v2/projects/<project_id>/branches/<branch_id>/data-api' \
+  --header 'Content-Type: application/json' \
+  --header 'Authorization: Bearer <token>' \
+  --data '{
+       "database_name": "only-needed-if-you-want-to-expose-more-than-one-database"
+  }'
+```
+
+Sample response:
+
+```json shouldWrap
+{
+  "url": "https://app-restless-salad-23184734.dpl.myneon.app"
+}
+```
+
+</TabItem>
+</Tabs>
+
+Once enabled, your Data API instance usually takes about **15 seconds** to boot up (the endpoint URL is returned instantly). The instance will automatically scale down to zero when your database is idle, and scale back up on demand — aligning with your serverless database.
+
+## API Authentication
+
+When you call the `/data-api` endpoint described above, we automatically provision [**Neon Auth**](/docs/guides/neon-auth) for that project. We set up the Data API's authentication to match this instance of Neon Auth.
+
+When you enable the Data API — either by calling the `/data-api` endpoint or from the Console — we automatically provision [**Neon Auth**](/docs/guides/neon-auth) for your project if it isn't already set up. If your project already has Neon Auth enabled, the Data API will re-use your existing configuration.
+
+This means you need to send a valid JWT from Neon Auth with every Data API request that is protected by RLS policies.
+
+### Third-party auth
+
+You can also bring your own **JWKS** from a third-party provider (like Clerk, Keycloak, Auth0, or Better Auth), adding more auth flexibility. Just include `jwks_url` (and optionally `jwks_audience`) in your request:
+
+```bash shouldWrap
+curl --location 'https://console.neon.tech/api/v2/projects/<project_id>/branches/<branch_id>/data-api' \
+  --header 'Content-Type: application/json' \
+  --header 'Authorization: Bearer <token>' \
+  --data '{
+       "jwks_url": "https://url.to.your/.well-known/jwks.json"
+  }'
+```
+
+## Using the Data API
+
+By default, all tables in your database are accessible via the API with `SELECT` permissions granted to **unauthenticated requests**. This lets you directly interact with the API without requiring additional authorization headers.
+
+<Admonition type="warning">
+We strongly recommend enabling Row Level Security (RLS) on _all_ tables as soon as you enable the Data API. You can do this with:
+  
+```sql
+ALTER TABLE <schema_name>.<table_name> ENABLE ROW LEVEL SECURITY;
```

Comment:
maybe add a comment on the line before saying `-- for every table, on every schema`